_docs/_latest/setup/install.md

```diff
@@ -42,31 +42,37 @@ steps below:
 
 ### Run setup
 
-  1. After you've started Cloud Shell, download Forseti. The installer is
-     included. Getting `2.0-dev-rc1` branch will install Forseti v2.0 Release
-     Candidate 1.
+  1. After you've started Cloud Shell, download Forseti. The installer is included. 
 
       ```bash
-      git clone -b master --single-branch https://github.com/GoogleCloudPlatform/forseti-security.git
+      git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+      ```
+
+  1. Checking out a specific version of Forseti by tag, e.g. `v2.0.0.`.
+
+      ```bash
+      # make sure you are in the forseti-security folder.
+      git fetch --all
+      git checkout tags/v2.0.0
       ```
 
   1. Running the installer:
 
      To install both client and server:
      
      ```bash 
-     python setup/installer.py
+     python install/gcp_installer.py
      ```
 
-      To see additional configurations for the setup:
+     To see additional configurations for the setup:
```

Comment:
Please disregard use of a colon before we show something, it's not needed.

Reply:
Please fix this everywhere in this doc, e.g. L90.

Reply:
Updated